aplib.net-demo/Assets/Scripts/WFC/GridPlacer.cs

```diff
@@ -2,7 +2,7 @@
 using System;
 using System.Collections.Generic;
 using UnityEngine;
-using UnityEngine.Serialization;
+using static Assets.Scripts.Tiles.Direction;
 using Random = System.Random;
```

Comment:
Should we perhaps use the UnityEngine.Random?

Reply:
Idk, this is not my code.